tests/cupy_tests/core_tests/test_core.py

```diff
@@ -1,11 +1,13 @@
 import unittest
+import sys
 
 import numpy
 import pytest
 
 import cupy
 from cupy.core import core
 from cupy import testing
+from tests.cupy_tests.core_tests import test_raw
```

Comment:
```suggestion
from cupy_tests.core_tests import test_raw
```

Reply:
Thanks, @kmaehashi. I feel absolute importing from other test modules is shaky...